update_manifest_installer_comptabilite_annuelle.exe

STENS66 opened the pull request:
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.9 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.9.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/200909)

STENS66 committed:
update_manifest_installer_comptabilite_annuelle.exe

stephengillie commented:
Hi @STENS66 ,

Manual Validation ended with:
![image](https://github.com/user-attachments/assets/ec0be81f-2685-46c5-bd14-62c8e6d38636)

Is another installer switch needed to accept the license agreement? Also, does this mean an `Agreements` section is needed?

STENS66 committed:
Update GaetanSencie.Comptabiliteannuelle.installer.yaml

Added the "Agreement" section to be able to accept the license terms before starting the installation.

STENS66 commented:
Thank you Stephen for your last comment. I do confirm that an "Agreement" section is necessary to be able to accept the terms of the license before starting the installation but I don't know if another switch is necessary. Following your last recommendation, I added a "Agreements" section. I hope that the implementation is correct and will finally allow this manifest to be validated.

STENS66 committed:
Update GaetanSencie.Comptabiliteannuelle.installer.yaml

Correction agreements section

stephengillie commented:
Output of `/?`:
![image](https://github.com/user-attachments/assets/d524b225-3c78-45d5-aa4d-f1f1dd28752f)
The window extends off the screen. I do not see any license option.

stephengillie commented:
Adding https://github.com/microsoft/winget-pkgs/labels/Blocking-Issue to prevent my automation from continually re-running.

ItzLevvie commented:
@wingetbot run

KarbitsCode commented on manifests/g/GaetanSencie/Comptabiliteannuelle/1.7.0.0/GaetanSencie.Comptabiliteannuelle.installer.yaml:
```diff
@@ -0,0 +1,28 @@
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /silent
+  SilentWithProgress: /passive
```
```suggestion
```

This `InstallerSwitches` is not needed since winget automatically assigns proper switches from `InstallerType: inno`

Manually validated in Windows Sandbox:

![Screenshot 2025-01-01 231408](https://github.com/user-attachments/assets/cde980b9-6686-4fec-94b2-55840957dbf3)

cc @stephengillie

STENS66 committed:
Update GaetanSencie.Comptabiliteannuelle.installer.yaml

Removed "InstallerSwitches" done as suggested by KarbitsCode.

STENS66 commented:
Thank you to your team for this latest review.

stephengillie commented:
Verify:
![image](https://github.com/user-attachments/assets/7295f746-deb2-49ec-a542-858adacd40cd)
I couldn't find how to have the application display its version number.
![image](https://github.com/user-attachments/assets/c3f49fea-c13b-4a14-909f-7732652e8d08)
